src/Symfony/Component/Translation/Tests/TranslatorTest.php

```diff
@@ -622,7 +650,6 @@ public function getInvalidLocalesTests()
     public function getValidLocalesTests()
     {
         return [
-            [''],
```

Comment:
not sure we should change this behavior in a patch release :/
the previous `?? \Locale::getDefault()` approach was fine no?

Reply:
Actually I am not sure too, but good point. 

I thought it would be good have both `Translator` and `TranslatorTrait` work the same way.

Reply:
> I thought it would be good have both Translator and TranslatorTrait work the same way.

i think that's worth pursuing yes ... on master, then we wait for @nicolas-grekas's review also :)